### PR TITLE
[ICP-12776] Zigbee Multi Button: fix multi events issue

### PR DESCRIPTION
**Pull request by ZouLianghuiS**
@tpmanley , @greens , @dkirker  please have a review of my PR, thanks

### CODE REVIEW

**Comment by tpmanley** [comment]
@ZouLianghuiS  can you rebase to remove the extra commits?

File: devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy [hunk 1]
```diff
@@ -76,9 +76,9 @@ def parseAttrMessage(description) {
 		map = parseAduroSmartButtonMessage(descMap)
     	} else if (descMap?.clusterInt == zigbee.ONOFF_CLUSTER && descMap.isClusterSpecific) {
 		map = getButtonEvent(descMap)
-	} else if(descMap?.clusterInt == 0xFC80) {
+	} else if(descMap?.clusterInt == 0x0005) {
```
Comment (dkirker) [inline devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: How does this affect the other buttons in this DTH?

File: devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy [hunk 2]
```diff
@@ -226,7 +230,13 @@ private getButtonMap() {[
...
 				"03" : 1,
 				"04" : 2
-		]
+		],
+        "SceneSwitch-EM-3.0" : [
+				"01" : 1,
+                "02" : 2,
+                "03" : 3,
+                "04" : 4
+        ]
```
Comment (dkirker) [inline devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: The spacing should be tabs and not spaces.

**Comment by ZouLianghuiS** [comment]
> @ZouLianghuiS can you rebase to remove the extra commits?

hi, sorry for neglect the commits, I've just rebased my PR,please check again

File: devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy (hunk 1 above)
Comment (ZouLianghuiS) [reply devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: the if branch for cluster int 0xFC80 was added by meifen months ago, specifically for this heiman multi-button device, after I changed configure function for the device, the clusterInt changed to 0x0005, so this statement only works for heiman multi-button, won't affect other devices

File: devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy (hunk 2 above)
Comment (ZouLianghuiS) [reply devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: made new commit to replace spaces, please review again

File: devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy (hunk 1 above)
Comment (dkirker) [reply devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: I guess the last thing I would check is to make sure that these two devices don't suddenly mis-behave:
```
fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", outClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", manufacturer: "AduroSmart Eria", model: "ADUROLIGHT_CSC", deviceJoinName: "Eria scene button switch V2.1", mnmn: "SmartThings", vid: "generic-4-button"
fingerprint inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", outClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FCCC, 1000", manufacturer: "ADUROLIGHT", model: "ADUROLIGHT_CSC", deviceJoinName: "Eria scene button switch V2.0", mnmn: "SmartThings", vid: "generic-4-button"
```

Also, nit: spacing between the `if` and `(`: `} else if(`

Comment (ZouLianghuiS) [reply devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy]: add space in new commit

and from the logic, the two devices will be handled in previous case:
else if (isAduroSmartRemote()) {//ADUROLIGHT_CSC is defined in isAduroSmartRemote
		map = parseAduroSmartButtonMessage(descMap)
    	}

**Review by greens — APPROVED** [review]
Looks like the changes done in https://github.com/SmartThingsCommunity/SmartThingsPublic/commit/253ca32f09cbf9b188b910079416f51aafe5e243

**Pull request merged**